backend/src/routes/v0/agent/__init__.py

```
@@ -1,12 +1,13 @@
 from fastapi import Query, Response, status, Depends, APIRouter, Path
 from fastapi.responses import JSONResponse
 from sqlalchemy.orm import Session
```

Ruff (0.11.9)

`sqlalchemy.orm.Session` imported but unused

(F401)